### ISSUE TITLE
Numbers in name

### ISSUE THREAD

**Issue by apuntovanini** [loeffel-io/ls-lint#10]
Hi, thanks, I love this, finally something to make file naming consistent
A question: 
```
./docs/5x1000_leads.rb failed for rules: snakecase
./db/migrate/20200109074618_create_dashboards.rb failed for rules: snakecas
```

Should it fail? Numbers for migration (i.e.) are necessary, i'd like to test them all anyways: how can it be solved?
Thanks!

**Comment by loeffel-io** [comment]
Hey @apuntovanini,

you simply can use the new regex rules for this case: 

```yaml 
ls:
  ...

  docs:
    .rb: regex:[a-z0-9\_]+

  db/migrate:
    .rb: regex:[a-z0-9\_]+
```

**Comment by apuntovanini** [comment]
Thanks! It works
Maybe I should file another issue (or I get something wrong), but multiple validations fail with a wrong error: 
```
failed for rules: regex ([a-z0-9\_]+), regex ([a-z0-9\_]+)
```
Plus, they seem to work only if a space is added after the comma in the yaml file
Thanks again
Andrea

**Comment by loeffel-io** [comment]
hey Andrea,

can you please post your `.ls-lint.yml` file here

Thanks!

**Comment by apuntovanini** [comment]
Sure

```yaml
ls:
  .dir: PascalCase, kebab-case, snake_case
  .css: kebab-case
  .ts: PascalCase, camelCase
  .tsx: PascalCase, camelCase
  .js: camelCase
  .png: kebab-case
  .rb: snake_case, regex:[a-z0-9\_]+
  .rake: snake_case, regex:[a-z0-9\_]+
  .jpeg: kebab-case, regex:[a-z0-9\-]+

  app/controllers:
    .dir: snake_case

  app/javascript:
    .dir: PascalCase, kebab-case

ignore:
  - test
  - tmp
  - spec
  - storage
  - public
  - node_modules
  - typings
  - build
  - app/config
  - babel.config.js
  - postcss.config.js
  - .yarn
  - .vscode
  - .git
  - app/javascript/__generated__

```

**Comment by loeffel-io** [comment]
Thanks, 

rules are logicly *and* combined, not *or* 

**Example:** 

```yaml
ls:
  .dir: PascalCase, kebab-case, snake_case
```

will check if the PascalCase **AND** kebab-case **AND** the snake_case rule matches your directory name. 

So this makes no sense

**Comment by loeffel-io** [comment]
I will add this information to the readme 👍

**Comment by loeffel-io** [comment]
Added 👍

**Comment by apuntovanini** [comment]
ok, thanks for the explanation!

But how would you check against:

```
ComponentName/index.tsx
ComponentName.tsx
```

and mark both as valid? Maybe it's just outside the scope of the library
Thanks!

**Comment by loeffel-io** [comment]
In this case you simply have to override your configuration for the directory `ComponentName` like:

```yaml
ls:
  .tsx: camelCase
  
  ComponentName: 
    .tsx: kebab-case # for example
```
